config.js

```diff
@@ -20,4 +20,7 @@ module.exports = {
   comma:   global.FX_STYLE_COMMA   || noop,
   colon:   global.FX_STYLE_COLON   || noop,
   list:    global.FX_STYLE_LIST    || list,
+
+  // set this to a filename to send debug/tracing output there
+  // log: '/tmp/fx.log',
```

Comment:
I this is not needed. fx can be debugged with `console.error` and `fx 2>err.log`